handle_http.go

```diff
@@ -596,30 +592,20 @@ Loop:
 				if resp.Duration() < time.Second*time.Duration(slowTransferRampupTime) {
 					continue
 				} else if startBelowLimit == 0 {
-					log.Warnln("Download speed of ", resp.BytesPerSecond(), "bytes/s", " is below the limit of", downloadLimit, "bytes/s")
+					//log.Warnln("Download speed of ", resp.BytesPerSecond(), "bytes/s", " is below the limit of", downloadLimit, "bytes/s")
 					startBelowLimit = time.Now().Unix()
 					continue
-				} else if (time.Now().Unix() - startBelowLimit) < slowTransferWindow {
+				} else if (time.Now().Unix() - startBelowLimit) < int64(slowTransferWindow) {
 					// If the download is below the threshold for less than `SlowTransferWindow` (default 30) seconds, continue
 					continue
 				}
 				// The download is below the threshold for more than `SlowTransferWindow` seconds, cancel the download
 				cancel()
 				if ObjectClientOptions.ProgressBars {
-					var cancelledProgressBar = p.AddBar(0,
-						mpb.BarQueueAfter(progressBar, true),
-						mpb.BarFillerClearOnComplete(),
-						mpb.PrependDecorators(
-							decor.Name(filename, decor.WC{W: len(filename) + 1, C: decor.DidentRight}),
-							decor.OnComplete(decor.Name(filename, decor.WCSyncSpaceR), "cancelled, too slow!"),
-							decor.OnComplete(decor.EwmaETA(decor.ET_STYLE_MMSS, 0, decor.WCSyncWidth), ""),
-						),
-						mpb.AppendDecorators(
-							decor.OnComplete(decor.Percentage(decor.WC{W: 5}), ""),
-						),
-					)
-					progressBar.SetTotal(resp.Size, true)
-					cancelledProgressBar.SetTotal(resp.Size, true)
+					progressBar.Abort(true)
+					log.Errorln("cancelled, too slow!")
+					log.Info("retrying...")
+					//cancelledProgressBar.SetTotal(resp.Size, true)
```

Comment:
Isn't there a higher function that emits an error message?  Seems strange to do that from the progress bar.